app/build.gradle

```diff
@@ -40,9 +40,9 @@ dependencies {
     implementation "androidx.multidex:multidex:$multidex_version"
     implementation "androidx.appcompat:appcompat:$appcompat_version"
     implementation "androidx.constraintlayout:constraintlayout:$constraintlayout_version"
+    implementation "androidx.preference:preference:$preference_version"
     implementation "com.google.android.material:material:$material_version"
     implementation "com.google.guava:guava:$guava_version"
-    implementation "com.squareup.picasso:picasso:$picasso_version"
```

Comment:
Why did we have Picasso? :)

Reply:
haha, I forgot to delete it from the previous version of the sample app. 😅